hugo/content/045_contribute/20_documentation/25_markup/attachments.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Attachments
+description : "The Attachments shortcode displays a list of files attached to a page."
+---
+
+The Attachments shortcode displays a list of files attached to a page.
+
+{{% attachments /%}}
```

Comment:
the list of attachments is rendered using a very small font, same problem for other lists below on the same page

Reply:
c2dcfa0318d3f98098bc662ff09bde2c364a30be